mentor_classifier/pyproject.toml

```diff
@@ -21,6 +21,7 @@ spacy = "^3.1.0"
 python-dotenv = "^0.19.0"
 boto3 = "^1.18.31"
 pymongo = {extras = ["srv"], version = "^3.12.0"}
+tomlkit = "^0.10.0"
```

Comment:
pip should install tomlkit when it installs poetry here https://github.com/mentorpal/mentor-classifier/blob/main/mentor_classifier/Dockerfile#L27, try it yourself

Reply:
poetry should just install classifier's dependencies, and afaik tomlkit is not one of them right?

Reply:
https://github.com/mentorpal/mentor-classifier/runs/5453650683?check_suite_focus=true#step:5:886

This is the error I get if I don't include it in poetry

Reply:
so i think i hit the same issue, on the first run, poetry removes tomlkit that pip installed. so if you want to fix that then just add pip install tomlkit before running poetry again

Reply:
Okay, I'll do some more investigation. FYI, the latest release of the classifier was cut from this branch.

Reply:
and then line 790 poetry install removes tomlkit:
```
Package operations: 60 installs, 8 updates, 2 removals
  • Removing pastel (0.2.1)
  • Removing tomlkit (0.7.2)
```
so after that poetry stops running!?

Reply:
one way to solve it is to call `pip install tomlkit pastel` before running poetry again. If you have time perhaps you can investigate why poetry removes tomlkit. 

Anyways, there's a bigger problem with the install, it does install both full and cpu-only version of torch! So you can remove poethepoet and then you'll only run poetry once.

Reply:
installing tomlkit after poetry worked

Reply:
Same thing was occuring for everytime poetry was ran, had to add `pip install tomlkit` after each `poetry install` in the dockerfiles for both mentor_classifier and mentor_classifer_api